internal/controller/kustomization_wait_test.go

```diff
@@ -325,6 +328,11 @@ data: {}
 		Spec: kustomizev1.KustomizationSpec{
 			Interval: metav1.Duration{Duration: 2 * time.Minute},
 			Path:     "./",
+			KubeConfig: &meta.KubeConfigReference{
+				SecretRef: meta.SecretKeyReference{
+					Name: "kubeconfig",
```

Comment:
Just curious, why was this needed?

Reply:
This is how all KC tests run, using the envTest kubeConfig generated at startup. Previously we were bypassing impersonation...